test/factories/host_related.rb

```diff
@@ -29,7 +29,7 @@ def set_environment_taxonomies(host_or_hostgroup, environment = host_or_hostgrou
 FactoryBot.define do
   factory :ptable do
     sequence(:name) { |n| "ptable#{n}" }
-    layout { 'zerombr\nclearpart --all    --initlabel\npart /boot --fstype ext3 --size=<%= 10 * 10 %> --asprimary\npart /     --f   stype ext3 --size=1024 --grow\npart swap  --recommended' }
+    layout { "zerombr\nclearpart --all    --initlabel\npart /boot --fstype ext3 --size=<%= 10 * 10 %> --asprimary\npart /     --fstype ext3 --size=1024 --grow\npart swap  --recommended" }
```

Comment:
Nit: I'd like this in separate commit bit better, but it's just me nitpicking...

Reply:
I have to correct it here because kickstart would fail to validate :)